examples/clone.rs

```diff
@@ -23,7 +23,7 @@ fn main() {
     let mut greeter = GreeterMock::new();
 
     greeter
-        .expect_greet(partial_eq("Tom"))
+        .expect_greet(|a| a.partial_eq("Tom"))
```

Comment:
I think the examples would be more inviting for a new user if the closure parameter was named `arg` instead of `a`

Reply:
Addressed in f22cd76fc97ee00643a49c7cec655076229458e1